src/pages/Cadastro/Cadastro.jsx

```diff
@@ -6,24 +6,30 @@ import Header from "../../common/Header/Header";
 import Footer from "../../common/Footer/Footer";
 
 export function Cadastro({ isOpen, onClose }) {
-  // const handleCadastro = async () => {
-  //   e.preventDefault()
+  const handleCadastro = async () => {
+    e.preventDefault();
 
-  //   const body = {
-  //     nomeCompleto,
-  //     email,
-  //     senha,
-  //     confirmaSenha,
-  //     fotoPerfil,
-  //     biografia,
-  //     linkedin,
-  //     profissao
-  //   }
+    const body = {
+      nomeCompleto,
+      email,
+      senha,
+      confirmaSenha,
+      fotoPerfil,
+      biografia,
+      linkedin,
+      profissao,
+    };
 
-  //   if (senha === confirmaSenha) {
-  //     const resposta = await postMentores(body)
-  //   }
-  // }
+    if (senha === confirmaSenha) {
```

Comment:
aqui podemos fazer a validacao inversa ex: `if (senha !== senhaConfimada)` e chamar a notificacao de erro + early return

Reply:
outra coisa, essas validacoes de formulario podem ir dentro do componente de formulario mesmo, dai la de dentro do componente vc só chama a prop function (`onSave` por exemplo) quando o form for valido